### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by SakshiShreya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,24 +4,32 @@
   "description": "",
   "main": "script.js",
   "scripts": {
-    "start": "nodemon --exec babel-node script.js",
-    "debug": "nodemon --inspect --exec babel-node script.js",
+    "dev": "nodemon --exec babel-node ./src",
```
Comment (SakshiShreya) [inline package.json:7]: run `npm start dev` from now on because heroku uses `npm start` for deployment.

File: package.json [hunk 2]
```diff
@@ -4,24 +4,32 @@
...
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Rohan Mittal",
   "license": "MIT",
   "dependencies": {
+    "@babel/cli": "^7.13.16",
+    "@babel/core": "^7.13.16",
+    "@babel/node": "^7.13.10",
+    "@babel/preset-env": "^7.13.10",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (SakshiShreya) [inline package.json:22]: Added babel in dependencies because heroku will use it for deployment

File: package.json [hunk 3]
```diff
@@ -4,24 +4,32 @@
...
-    "@babel/core": "^7.13.10",
-    "@babel/node": "^7.13.10",
-    "@babel/preset-env": "^7.13.10",
-    "babel-core": "^7.0.0-bridge.0",
     "nodemon": "^2.0.7",
     "regenerator-runtime": "^0.13.7"
+  },
+  "engines": {
+    "node": "12.x"
   }
```
Comment (SakshiShreya) [inline package.json:34]: Added engines for heroku to install node accordingly.

**Review by SakshiShreya — COMMENTED** [review]
.

**Review by rohanmittal149 — APPROVED** [review]

**Pull request merged**